tests/containers/base_image_test.py

```diff
@@ -9,7 +9,7 @@
 import re
 import tempfile
 import textwrap
-from typing import TYPE_CHECKING, Any, Callable
+from typing import TYPE_CHECKING, Any, Callable, List
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Any, Callable
```

you can use `list[str]` as the type, no need to import the capital L list

this is a new feature beginning with some version of python, but we do have it available here

Reply:
you're right, I'll update